src/components/header/styles.scss

```diff
@@ -1,18 +1,37 @@
 @import '~styles/settings';
 
 .c-header {
+  position: fixed;
   width: 100%;
-  height: 50px;
+  height: 55px;
   font-family: $font-family;
   font-size: 14px;
   display: flex;
   align-items: center;
   padding: 0 50px;
+  background: $color-white;
+  background: linear-gradient(180deg, $color-white 0%, transparent 100%);
+  z-index: $z-index-4;
 
-  .icon {
-    width: 20px;
-    max-height: 40px;
-    fill: $color-gray-1;
+  .menu-button {
+    height: 40px;
+    position: absolute;
+    font-size: 20px;
```

Comment:
let's talk about text styles, this is what I refer to when I talk about custom styles, better to have a closed API where classes cant define stuff such as `text-transform`, `font-size`, `font-weight` and `color`